tools/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+if(UNIX)
+    install(
+        PROGRAMS
+            dvloupe
+            dvmap
+            dvpackager
+            dvplay
+            dvsampler
+        DESTINATION bin
+        )
+endif()
```

Comment:
What is the modernity difference with 
`if(UNIX)
  install(PROGRAMS ${dvrescue_TOOLS} DESTINATION bin)
endif()` (current version)?

Reply:
The modernity difference is that it is declarative and doesn't use variable expansion. CMake is more maintainable if it is declarative.